.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+
+jobs:
+  fedora:
+    name: Linux Fedora (Release, GCC, cxx11)
+    runs-on: ubuntu-latest
+    container: dynawo/dynawo-ci:latest
+    strategy:
+      fail-fast: false
+    env:
+      DYNAFLOW_LAUNCHER_BUILD_TYPE: Release
+      DYNAFLOW_LAUNCHER_PROCESSORS_USED: 2
+      DYNAFLOW_LAUNCHER_FORCE_CXX11_ABI: "true"
+      DYNAFLOW_LAUNCHER_LOCALE: "en_GB"
+      COMPILER_FOLDER: "gcc5.3.1"
+      ZIP_ALGORITHMS: "dynawo-algorithms-fedora-Release-GCC-cxx11.zip"
+      ZIP_DYNAWO: "dynawo-fedora-Release-GCC-cxx11.zip"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch Dynawo
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: "dynawo/dynawo"
+          version: "tags/nightly"
+          file: "${{ env.ZIP_DYNAWO }}"
+          target: "dynawo/dynawo.zip"
+          token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Extract Dynawo
+        run: |
+          cd dynawo
+          unzip -qq "dynawo.zip"
+          cd ..
+
+      - name: Fetch Dynawo algorithms
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: "dynawo/dynawo-algorithms"
+          version: "tags/nightly"
+          file: "${{ env.ZIP_ALGORITHMS }}"
+          target: "dynawo-algorithms/dynawo-algorithms.zip"
+          token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Extract Dynawo algorithms
+        run: |
+          cd dynawo-algorithms
+          unzip -qq "dynawo-algorithms.zip"
+          cd ..
+
+      - name: Load persistent build files
+        uses: actions/cache@v2
+        with:
+          # A list of files, directories, and wildcard patterns to cache and restore
+          path: |
+            build
```

Comment:
Same remark for cache as the two other projects

Reply:
Cache has been removed